lib/routes/content.js

```diff
@@ -126,14 +164,15 @@ const routes = application => {
      * @param {object} res - HTTP Response
      * @param {object} next - Express callback
     */
-    app.get(`/${config.content.base}/:type/:id/:revision/${config.content.actions.edit}`, (req, res, next) => {
+    app.get(`/${config.content.base}/:type/:revision/${config.content.actions.edit}`, (req, res, next) => {
```

Comment:
Needs to be `/:id/:revision` (I know we only need to look up by revision but we need the ID for user s to be able to get to the listing of all IDs. REST architecture. Applies to workflows too)